front/src/modules/conflict/components/ConflictsListV2.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useMemo } from 'react';
+
+import cx from 'classnames';
+import { useTranslation } from 'react-i18next';
+
+import type { ConflictV2 } from 'common/api/osrdEditoastApi';
+import type { TrainScheduleWithDetails } from 'modules/trainschedule/components/TimetableV2/types';
+
+import ConflictCardV2 from './ConflictCardV2';
+import type { ConflictWithTrainNames } from '../types';
+import addTrainNamesToConflicts from '../utils';
+
+type ConflictsListV2Props = {
+  conflicts: ConflictV2[];
+  expanded: boolean;
+  trainSchedulesDetails: TrainScheduleWithDetails[];
+  toggleConflictsList: () => void;
+  onClick: (conflict: ConflictWithTrainNames) => void;
```

Comment:
```suggestion
  onConflictClick: (conflict: ConflictWithTrainNames) => void;
```

Reply:
Changing the name of on click disables its use, so I have to leave it as is

Reply:
I don't understand, it works well for me.